src/content/peerconnection/negotiate-timing/js/main.js

```diff
@@ -72,6 +80,7 @@ function getOtherPc(pc) {
 async function start() {
   console.log('Requesting local stream');
   startButton.disabled = true;
+  startButtonPlanB.disabled = true;
```

Comment:
hm. this is ugly (when we have start buttons, code referring to them should go with the button-specific thing), but startButton and startButtonPlanB have to know about each other, so perhaps this is the best way to write it. No change needed.